bundles/configuration.rst

```diff
@@ -46,13 +47,13 @@ as integration of other related components:
 .. sidebar:: Using Parameters to Configure your Bundle
 
     If you don't have plans to share your bundle between projects, it doesn't
```

Comment:
Then don't create a bundle...?

Reply:
Haha, yea, I agree with Wouter. We can still say this is optional, of course, but if you're reading this, then you *do* want to share your bundle between projects :)

Reply:
You are completely right! This only made sense in the past, where bundles could be non-shareable ... but that's no longer the case. Removed the entire sidebar!